release notes/v1.2.0.md

```diff
@@ -0,0 +1,363 @@
+k6 v1.2.0 is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+## Breaking changes
+
+- The experimental Open Telemetry and Prometheus outputs are now enforcing TLSv1.3. This should've been the default to begin with. It is not expected that anyone should be affected, apart from making it more secure for the metrics output to send messages.
```

Comment:
```suggestion
- The experimental Open Telemetry and Prometheus outputs now default to TLSv1.3. This should've been the default to begin with. It is not expected that anyone should be affected, apart from making it more secure for the metrics output to send messages.
```
As someone not involved in this change, enforcing it generates doubts for me. What does it mean? Is it just a default value, and can I configure it?